website/playground/Playground.js

```diff
@@ -30,7 +30,13 @@ const ENABLED_OPTIONS = [
   "useTabs",
   "semi",
   "singleQuote",
-  "bracketSpacing",
+  // [prettierx]: broken-out bracket spacing options
+  "graphqlCurlySpacing",
+  "exportCurlySpacing",
+  "importCurlySpacing",
+  "objectCurlySpacing",
+  "typeCurlySpacing",
+  "yamlBracketSpacing",
```

Comment:
Interesting, did you try building this playground?

Reply:
I didn't - but then I did to see if it'd work. With a few small changes it did:

![image](https://user-images.githubusercontent.com/9868643/120565004-febf0200-c3c0-11eb-8669-fcaa87ba786c.png)

However, I think getting the playground/site to work may be beyond the scope of this PR. On the one hand, there are changes to the site upstream. But I'm not certain that's sufficient cause. Should we omit the changes to the site entirely, then, from this PR, and separate them into their own?